frontend/src/NewRidinglessonCard.tsx

```diff
@@ -49,11 +53,11 @@ export default function NewRidinglessonCard(props: Props): JSX.Element {
     }
     return(
         <div className="ridinglesson-card new-ridinglesson">
-            <input type="teacher" onInput={changeteacher}/>
-            <input type="type" onInput={changetype}/>
-            <input type="instructor" onInput={changeinstructor}/>
-            <input type="date" onInput={changeDate}/>
-            <input type="time" onInput={changeTime}/>
+            <input type="teacher" value={teacher} onInput={changeteacher}/>
```

Comment:
es gibt im html keinen type teacher oder instructor, ich vermute mal du meinst text.
des Weiteren könntest du satt immer eine Function aufzurufen, folgendes eingeben. 
```suggestion
            <input type="text" value={teacher} onInput={event => setTeacher(event.target.value)}/>
```